### ISSUE TITLE
Add plotting routine for creating "Manhattan" plots

### ISSUE THREAD

**Issue by stufield** [SomaLogic/SomaPlotr#13]
- create a "Manhattan" plotting routing for package
- there is an existing routine from our internal code base, called `plotManhattan()` the could be used as a starting point.
- doesn't have to be identical, but could act as a basis
- we would have to figure out how to generate the effect sizes/p-values or (similar to `plotVolcano()`) have the user provide those as an argument, this is probably preferable, because calculating statistics is out of scope for this plotting package

**Comment by amanda-hi** [comment]
Our internal `plotManhattan()` uses a lot of other functions that will not be available to SomaPlotr, so I think I can use the ggplot2 code as a starting point, but will need to ditch all the pre-processing steps. I'll have to start w/ SomaDataIO's `example_data` and re-create the data processing steps, without using any internal (and currently publicly unavailable) functions. I'm starting on this now, but it might take a bit of work to re-create and/or simplify everything the internal functions do.

**Comment by stufield** [comment]
Yes, I think you'll have to enter the workflow a little more downstream that the existing plot (which is fine, probably better) since you shouldn't really be doing any statistics "under the hood" of a plotting routine anyhow. So the input for the plot would be the output of the statistical analysis. This part we could do in the examples or vignettes. I think `plotVolcano()` already does this, or something similar, is that correct?
